apps/server/src/routes/project.ts

```
@@ -0,0 +1,193 @@
+			id: lyricsId,
+			createdAt: now,
+			updatedAt: now,
+			text: updates.lyrics.text || '',
+			projectId: id,
+			lines: updates.lyrics.lines || [],
+		};
+
+		await c.env.LYRICS_KV.put(`lyrics:${lyricsId}`, JSON.stringify(lyrics));
+
+		// Remove lyrics from updates to avoid storing in project
+		delete updates.lyrics;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)